### DISCUSSION TITLE
Binance chain namespace setup

### DISCUSSION THREAD

**Discussion by downing034** [WalletConnect/walletconnect-monorepo#2466] [Developer Support]
We currently use the sign-client to connect to our dapp. Our dapp connects to the OG binance chain which I believe is cosmos based (not binance smart chain).

I tried setting it up inside the cosmos namespace, but received: `unsupported or empty chains for namespace` using chain id `Binance-Chain-Ganges`. Is there a different namespace this should be included under or does this chain need to be added to the list of supported cosmos chains?

```
const namespaces = {
  cosmos: {
      methods: Object.values(DEFAULT_COSMOS_METHODS),
      chains: ['cosmos:Binance-Chain-Ganges'],
      events: ['chainChanged', 'accountsChanged'],
  },
};
```

**Comment by finessevanes** [comment]
The chain will have to be added. See [Cosmos Testcases](https://github.com/ChainAgnostic/namespaces/blob/main/cosmos/caip-2.md#test-cases).

**Reply by downing034** [reply]
@finessevanes Just tried this out and it also throws the same error:
```
const namespaces = {
  cosmos: {
      methods: Object.values(DEFAULT_COSMOS_METHODS),
      chains: ['cosmos:Binance-Chain-Ganges'],
      events: ['chainChanged', 'accountsChanged'],
  },
};
```

**Reply by finessevanes** [reply]
That's the same chain. Try:

```javascript
const namespaces = {
  cosmos: {
      methods: Object.values(DEFAULT_COSMOS_METHODS),
      chains: ['cosmos:Binance-Chain-Tigris'],
      events: ['chainChanged', 'accountsChanged'],
  },
};
```
